addon/styles/app.css

```diff
@@ -43,3 +43,34 @@ vertical-item::after {
   clear: both;
   position: relative;
 }
+
+/* these classes are used as a visual aid for debugging edge calculations */
```

Comment:
It might be better to style the divs dynamically so that this code isn't included in production builds. Worth changing?